README.md

```diff
@@ -7,6 +7,10 @@ slaves is in the https://github.com/openshift/jenkins repository.
 For more information about using these images with OpenShift, please see the
 official [OpenShift Documentation](https://docs.openshift.org/latest/using_images/other_images/jenkins.html).
 
+The images are available to be pulled from https://access.redhat.com/containers/#/  
+For example the dotnet-22-jenkins-slave-rhel7 image is available at 
+https://access.redhat.com/containers/?tab=overview#/registry.access.redhat.com/dotnet/dotnet-22-jenkins-slave-rhel7
```

Comment:
It's a direct link to 2.2 - I'd rather not link to deprecated images, and I'd rather not introduce a version here that we would have to maintain. I think that the first line is enough as it takes one to a search bar. (And I'd remove the `/#/` at the end :P